sdk/Pulumi.Tests/Provider/PropertyValueTests.cs

```diff
@@ -317,6 +351,25 @@ public async Task DeserializingWrappedSecretInOutputWorks()
         Assert.True(data.IsKnown);
     }
 
+
+    [Fact]
+    public async Task DeserializingWrappedUnknownSecretInOutputWorks()
+    {
+        var serializer = CreateSerializer();
+        // secretOutput = Output(Secret(Unknown))
```

Comment:
I think these comments caused me more confusion than understanding, I'd just take them out. If you want a description here put it in English "secretOutput is Unknown wrapped in a Secret wrapped in an Output"